spec/features/welcome_spec.rb

```diff
@@ -14,17 +14,6 @@
     it 'displays the count of available artworks' do
       expect(page.body).to include '123 artworks available'
     end
-    {
```

Comment:
did we have to remove this?

Reply:
It's failing and I don't understand why. The UI looks fine.